recipe/bld.bat

```diff
@@ -16,3 +16,4 @@ if errorlevel 1 exit /b 1
 
 copy "%LIBRARY_PREFIX%"\bin\tiff.dll "%LIBRARY_PREFIX%"\bin\libtiff.dll
 copy "%LIBRARY_PREFIX%"\bin\tiffxx.dll "%LIBRARY_PREFIX%"\bin\libtiffxx.dll
+copy "%LIBRARY_PREFIX%"\lib\tiff.lib "%LIBRARY_PREFIX%"\lib\libtiff.lib
```

Comment:
I think we had that in the past but I'm not sure. @isuruf do you foresee any possible issues with this?

Reply:
that's fine